go/install/install_windows.go

```diff
@@ -208,6 +208,50 @@ func WatchdogLogPath(logGlobPath string) (string, error) {
 	return logName, err
 }
 
+const autoRegPath = `Software\Microsoft\Windows\CurrentVersion\Run`
+const autoRegOldName = `electron.app.keybase`
+const autoRegName = `Keybase.Keybase.GUI`
+
+func autostartStatus() (bool, error) {
+	k, err := registry.OpenKey(registry.CURRENT_USER, autoRegPath, registry.QUERY_VALUE|registry.READ)
+	if err != nil {
+		return false, fmt.Errorf("Error opening Run registry key: %v", err)
+	}
+	defer k.Close()
+
+	// Value not existing means that we are not starting up by default!
+	_, _, err = k.GetStringValue(autoRegName)
+	return err == nil, nil
+}
+
+func ToggleAutostart(context Context, on bool, forAutoinstallIgnored bool) error {
+	k, err := registry.OpenKey(registry.CURRENT_USER, autoRegPath, registry.QUERY_VALUE|registry.WRITE)
+	if err != nil {
+		return fmt.Errorf("Error opening StartupFolder registry key: %v", err)
+	}
+	defer k.Close()
+
+	// Delete old key if it exists.
+	k.DeleteValue(autoRegOldName)
```

Comment:
this can be removed once no one is running whatever client version is before this branch (5.2.0 i guess)? assuming that's true, can we rename this to something like `autoRegDeprecatedName` and leave a comment about when it can be deleted? or just pick a date. like, if it's 2022 and you see this, you can delete this.